labshare/templatetags/reservations.py

```diff
@@ -9,9 +9,21 @@ def current_reservation(gpu):
         return ""
     return reservations.order_by("time_reserved").first().user
 
+
 @register.filter
 def next_reservation(gpu):
     reservations = gpu.reservations.all()
     if len(reservations) <= 1:
         return ""
     return reservations.order_by("time_reserved").all()[1].user
+
+
+@register.filter
+def queue_position(gpu, user):
+    reservations = gpu.reservations.order_by("time_reserved").all()
+    user_reservations = reservations.filter(user__id=user.id)
```

Comment:
you could squash these two lines together like so: 
`reservations = gpu.reservations.order_by("time_reserved").filter(user__id=user.id)`
or the other way round:
`reservations = gpu.reservations.filter(user__id=user.id).order_by("time_reserved").`